packages/a11y/package.json

```diff
@@ -18,7 +18,7 @@
     "url": "https://github.com/WordPress/packages/issues"
   },
   "dependencies": {
-    "@wordpress/dom-ready": "^0.1.0-beta.6"
+    "@wordpress/dom-ready": "0.1.0-beta.6"
```

Comment:
This change shouldn't be in this PR should it?

Reply:
Likewise the `dom-ready/package-lock.json` and `a11y/package-lock.json` changes shouldn't be in this PR either?

Reply:
Yes, I will remove those changes.